internal/pkg/manifest/validate.go

```diff
@@ -909,3 +982,44 @@ func buildDependencyGraph(opts validateDependenciesOpts) (*graph.Graph, error) {
 	}
 	return dependencyGraph, nil
 }
+
+// Validate that paths contain only an approved set of characters to guard against command injection.
+// We can accept 0-9A-Za-z-_.
+func validatePath(input string) error {
```

Comment:
can we make the regex and this function name more specific? is this the `http` path or `volumes` path it's not clear 🥺